OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -52,10 +52,13 @@ public class HarvesterInfo : ITraitInfo, Requires<MobileInfo>
 		public readonly bool SearchOnCreation = true;
 
 		[Desc("Initial search radius (in cells) from the refinery that created us.")]
-		public readonly int SearchFromProcRadius = 24;
+		public readonly int SearchFromProcRadius = 30;
 
 		[Desc("Search radius (in cells) from the last harvest order location to find more resources.")]
-		public readonly int SearchFromOrderRadius = 12;
+		public readonly int SearchFromOrderRadius = 15;
+
+		[Desc("Adds this value to search radius when owner is not a human player.")]
```

Comment:
I feel the description should mention this doesn't apply to bots like you have said in your PR comment.